src/protections/NsfwProtection.ts

```diff
@@ -45,71 +45,62 @@ export class NsfwProtection extends Protection {
     }
 
     public async handleEvent(mjolnir: Mjolnir, roomId: string, event: any): Promise<any> {
-        if (event["type"] === "m.room.message") {
-            let content = JSON.stringify(event["content"]);
-            if (!content.toLowerCase().includes("mxc")) {
-                return;
-            }
-            // try and grab a human-readable alias for more helpful management room output
-            const maybeAlias = await mjolnir.client.getPublishedAlias(roomId);
-            const room = maybeAlias ? maybeAlias : roomId;
+        if (event.type !== "m.room.message" || event.type !== "m.sticker") {
+            return;
+        }
 
-            const mxcs = content.match(/(mxc?:\/\/[^\s'"]+)/gim);
-            if (!mxcs) {
-                //something's gone wrong with the regex
-                await mjolnir.managementRoomOutput.logMessage(
-                    LogLevel.ERROR,
-                    "NSFWProtection",
-                    `Unable to find any mxcs in  ${event["event_id"]} in ${room}`,
-                );
-                return;
-            }
+        const content = JSON.stringify(event.content);
+        const mxcs = content.match(/(mxc:\/\/[^\s'"]+)/gim);
```

Comment:
I removed the `c?` because I couldn't see the value in matching on `mx://`?

Reply:
that was probably an artifact of me cobbling together a regex